compiler/src/iree/compiler/Tools/iree_compile_lib.cc

```diff
@@ -49,10 +50,7 @@ enum class CompileMode {
 }  // namespace mlir
 
 int mlir::iree_compiler::runIreecMain(int argc, char **argv) {
-  llvm::setBugReportMsg(
-      "Please report issues to https://github.com/iree-org/iree/issues and "
-      "include the crash backtrace.\n");
-  llvm::InitLLVM y(argc, argv);
+  InitIree y(argc, argv);
```

Comment:
Ah the single-character class :-P Do you know the origin of that particular idiom? I think giving it a name like "unused" would at least be clearer

Reply:
Not sure where the y came from ... It's pretty old to have X https://github.com/llvm/llvm-project/commit/76d4632d92342a6776d58bc4ed4147f8ae9d05b7 and then Y https://github.com/llvm/llvm-project/blob/4b2a6e200f0836e7fcbc3f8a7416648c2b2929d0/llvm/tools/opt/opt.cpp#L347 and given you mostly have these inside a small function where you can see the entire scope, short name that you'd normally use for anything got picked.